english/eat-rainbows/right/right.md

```diff
@@ -2,4 +2,12 @@ You turned right and slammed into another customer carrying a birthday cake.
 
 Ughh, cake from Safeway with lard icing!
 
-Worst. Day. Ever.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.